src/schema/project.ts

```diff
@@ -16,6 +16,13 @@ export const projectCategoryEnum = pgEnum("project_category", [
   "ITT, Vehicles and Equipment",
 ]);
 
+export const projectFundStageEnum = pgEnum("project_fund_stage", [
```

Comment:
This is a nitpick. But, we can move this to the `project-fund` schema file

Reply:
Copy that, you specifically asked for it in the project file though lol

Reply:
Yeah... I failed to realize there would be a `project fund` schema file where it would be more appropriate. Thank you for being flexible!